### DISCUSSION TITLE
电脑发送手机文件，手机扫码出现404

### DISCUSSION THREAD

**Discussion by 2427825952** [alley-rs/fluxy#229] [Q&A] [answered]
电脑发送手机文件，手机扫码出现404
![3a813c24fb93c9033598e12076461a3](https://github.com/user-attachments/assets/2611d999-9f06-4178-92d5-0795f2cc8fd4)

**Comment by thep0y** [comment]
未遇到过此问题，请添加更多有用的信息。

**Reply by thep0y** [reply, marked as answer]
0.1.11版本已解决
